cmd/argchecker.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func verifyArguments(cmd *cobra.Command) error {
+	gwString, _ := cmd.Flags().GetString("gateway")
+	dbString, _ := cmd.Flags().GetString("dashboard")
+
+	if gwString == "" && dbString == "" {
```

Comment:
You are checking for both the parameters to be there but throwing an error that requires only one of them. Which one of the two is incorrect?